config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-gce.yaml

```diff
@@ -1,9 +1,23 @@
+presets:
+- labels:
+    preset-ci-gce-device-plugin-gpu: "true"
+  env:
+  - name: KUBE_FEATURE_GATES
+    value: DevicePlugins=true
+  - name: KUBE_GCE_NODE_IMAGE
+    value: gke-1-8-4-gke-1-cos-stable-63-10032-71-0-p-v171208-pre-nvda-gpu
```

Comment:
In a later PR, I would like to see this updated to a newer image, maybe `gke-1113-gke18-cos-stable-69-10895-91-0-v181115-pre-nvda-gpu`. But I don't have cycles to test this change.  cc @yguo0905 @jiayingz

Reply:
(I'll leave it to you to follow up :-) )